mlops_proj/src/dataset/watch_log.py

```
@@ -0,0 +1,75 @@
+import os
+
+import pandas as pd
+import numpy as np
```

Ruff (0.12.2)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)